src/packages/@ncigdc/modern_components/GeneExpression/index.ts

```diff
@@ -0,0 +1,4 @@
+import EnhancedGeneExpression from './EnhancedGeneExpression';
+// import createRenderer from './GeneExpression.relay';
```

Comment:
Is this foreseeing future implementation?

Reply:
indeed! I cribbed from clinical analysis and commented out things that aren't being used. once there's a graphQL request we'll need the renderer.